pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentStickyKeyDispatcherMultipleConsumers.java

```diff
@@ -215,13 +219,19 @@ protected void sendMessagesToConsumers(ReadType readType, List<Entry> entries) {
         final Map<Consumer, Set<Integer>> consumerStickyKeyHashesMap = new HashMap<>();
 
         for (Entry entry : entries) {
-            int stickyKeyHash = getStickyKeyHash(entry);
+            final int stickyKeyHash = getStickyKeyHash(entry);
+            final long ledgerId = entry.getLedgerId();
+            final long entryId = entry.getEntryId();
             Consumer c = selector.select(stickyKeyHash);
             if (c != null) {
                 groupedEntries.computeIfAbsent(c, k -> new ArrayList<>()).add(entry);
                 consumerStickyKeyHashesMap.computeIfAbsent(c, k -> new HashSet<>()).add(stickyKeyHash);
+                unMatchConsumerBackoff.reset();
             } else {
-                addMessageToReplay(entry.getLedgerId(), entry.getEntryId(), stickyKeyHash);
+                topic.getBrokerService().executor().schedule(() -> {
+                    addMessageToReplay(ledgerId, entryId, stickyKeyHash);
+                }, unMatchConsumerBackoff.next(), TimeUnit.MILLISECONDS);
```

Comment:
`unMatchConsumerBackoff.next()` seems odd here.
Consider the case when all entry in `entries` have the same `stickyKeyHash`, the delay time increases in the same batch entries.